projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/memberships.php

```diff
@@ -353,12 +356,15 @@ public function update_product( \WP_REST_Request $request ) {
 		$buyer_can_change_amount = isset( $request['buyer_can_change_amount'] ) && (bool) $request['buyer_can_change_amount'];
 
 		$payload = array(
-			'title'                   => $request['title'],
-			'price'                   => $request['price'],
-			'currency'                => $request['currency'],
-			'buyer_can_change_amount' => $buyer_can_change_amount,
-			'interval'                => $request['interval'],
-			'type'                    => $type,
+			'title'                        => $request['title'],
+			'price'                        => $request['price'],
+			'currency'                     => $request['currency'],
+			'buyer_can_change_amount'      => $buyer_can_change_amount,
+			'interval'                     => $request['interval'],
+			'type'                         => $type,
+			'welcome_email_content'        => $request['welcome_email_content'],
+			'subscribe_as_site_subscriber' => $request['subscribe_as_site_subscriber'],
+			'multiple_per_user'            => $request['multiple_per_user'],
```

Comment:
It looks perfect. 
As a side note, I usually don't like duplicated code, and I would have refactored some of that. I know some prefer explicit code to prevent complexity.

What is your point-of-view about that? (no need to change anything here, this is perfectly fine)